jobs/bbr-etcd/templates/backup.erb

```diff
@@ -1,16 +1,20 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
 set -euo pipefail
 
 source /var/vcap/jobs/etcd/bin/utils.sh
 
-BBR_ARTIFACT_FILE_PATH="${BBR_ARTIFACT_DIRECTORY}/snapshotdb"
-
+export BBR_ARTIFACT_FILE_PATH="${BBR_ARTIFACT_DIRECTORY}/snapshotdb"
```

Comment:
Why does this need to be exported?

Reply:
You are right, it does not need to be exported.

Reply:
@henderjm can you please confirm if [this commit](https://github.com/cloudfoundry-incubator/cfcr-etcd-release/pull/10/commits/749d0cfcc2122390a1f6105929a58e899a09b4f5) resolves this comment?